icu4j/main/core/src/test/java/com/ibm/icu/dev/test/UnicodeMapTest.java

```diff
@@ -27,6 +27,7 @@
 import org.junit.runners.JUnit4;
 
 import com.ibm.icu.impl.UnicodeMap;
+import com.ibm.icu.impl.UnicodeMap.EntryRange;
```

Comment:
optional: slight preference for not importing this

Reply:
Done